addons/mail/static/tests/new/composer/composer_tests.js

```diff
@@ -762,8 +762,7 @@ QUnit.test(
         await afterNextRender(() =>
```

Comment:
This `afterNextRender` is not removed on purpose?

Reply:
No I forgot to do it